Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -82,7 +97,9 @@ open class InputTextView: UITextView, AppearanceProvider {
         )
         placeholderLabel.pin(anchors: [.centerY], to: self)
         
-        isScrollEnabled = false
+        heightConstraint = heightAnchor.constraint(equalToConstant: minimumHeight)
+        heightConstraint?.isActive = true
+        isScrollEnabled = true
```

Comment:
Why setting `scrollEnabled` to true? Can you maybe add comment? From this context of PR I don't get it 😅 Also why this is not set on `setTextViewHeight` according to height? Or TextView handles this on it's own?

Reply:
The textView handles this internally, depending on its height and the content's height. Hence, we do not need to modify it depending on the conditions